tools/change_sr_commitment.php

```diff
@@ -22,9 +22,9 @@
 *
 ****************************************************************************************/
 
-$projectid = get_projectID_param($_POST, 'projectid');
-$action = get_enumerated_param($_POST, 'action', null, ['commit', 'withdraw']);
-$refresh_url = @$_POST['next_url'];
+$projectid = get_projectID_param($_REQUEST, 'projectid');
```

Comment:
I'm not a fan of making POST inputs into more general REQUEST inputs. This muddies the water on where scripts can legally get their data from and is not a pattern I want to establish.